acceptance/openstack/dns/v2/zones.go

```diff
@@ -0,0 +1 @@
+package v2
```

Comment:
Ah - OK, thanks.

If it's OK with you, this file gets renamed to `dns.go` in the next PR which follows the pattern of the other acceptance tests I've done. It also has actual contents.

No issues renaming it to `pkg.go` in this PR if you prefer, though.